var/spack/repos/builtin/packages/amdlibm/package.py

```diff
@@ -37,8 +35,8 @@ class Amdlibm(SConsPackage):
     depends_on("scons@3.1.2:", type=("build"))
     depends_on("mpfr", type=("link"))
 
-    patch('0001-libm-ose-Scripts-cleanup-pyc-files.patch')
-    patch('0002-libm-ose-prevent-log-v3.c-from-building.patch')
+    patch("0001-libm-ose-Scripts-cleanup-pyc-files.patch", when="@2.2")
```

Comment:
Is there a version before `2.2` that could be installed?  If so, then you'd want to use `@:2.2`.